src/runtime/components/data/Table.vue

```diff
@@ -250,12 +250,12 @@ export default defineComponent({
     function selectAllRows () {
       props.rows.forEach((row) => {
         // If the row is already selected, don't select it again
-        if (selected.value.some((item) => compare(toRaw(item), toRaw(row)))) {
+        if (isSelected(row)) {
```

Comment:
Are you sure calling `isSelected` is relevant here? There is a check for `modelValue` there.

Reply:
It is one extra check which should always be true at this point, shouldn't it ?  Just liked the readability but I can see your point. Would you prefer to have it reverted ?

Reply:
No actually you're right, `selected.value` doesn't exist anyway if `props.modelValue` is not defined.